content/en/docs/reference/command-line-tools-reference/kubelet-tls-bootstrapping.md

```diff
@@ -10,45 +10,160 @@ content_template: templates/concept
 
 {{% capture overview %}}
 
-This document describes how to set up TLS client certificate bootstrapping for
-kubelets. Kubernetes 1.4 introduced an API for requesting certificates from a
-cluster-level Certificate Authority (CA). The original intent of this API is to
-enable provisioning of TLS client certificates for kubelets. The proposal can be
+In a Kubernetes cluster, the components on the worker nodes - kubelet and kube-proxy - need to communicate with Kubernetes master components, specifically kube-apiserver.
+In order to ensure that communication is kept private, not interfered with, and ensure that each component of the cluster is talking to another trusted component, we strongly
+recommend using client TLS certificates on nodes.
+
+The normal process of bootstrappinng these components, and especially worker nodes which need certificates so they can communicate safely with kube-apiserver,
+can be a challenging process, often outside of the scope of Kubernetes, and requiring significant additional work. This, in turn, can make it challenging to
+initialize or scale a cluster.
```

Comment:
Suggestion: Made a few minor tweaks

"The normal process of bootstrappinng these components, especially worker nodes that need certificates so they can communicate safely with kube-apiserver, can be a challenging process as it is often outside of the scope of Kubernetes and require significant additional work. This in turn, can make it challenging to initialize or scale a cluster."

Reply:
Updated. Also cleaned up some minor typo and grammatical/syntactical issues.